### PR TITLE
add the relevant openshift template

### PR DESCRIPTION
**Pull request by andreaTP**

### CODE REVIEW

File: distro/openshift-template/mt/tenant-manager-template.yaml [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+- name: KEYCLOAK_URL
+  description: Keycloak base url
+  value: "http://localhost:8090"
+- name: KEYCLOAK_REALM
+  description: Keycloak realm
+  value: "apicurio-registry"
+- name: KEYCLOAK_API_CLIENT_ID
+  description: client id for Apicurio Registry API
+  value: registry-api
+- name: CLIENT_CREDENTIALS_BASIC_AUTH_ENABLED
```
Comment (carlesarnal) [inline distro/openshift-template/mt/tenant-manager-template.yaml:348]: This parameter can be removed.

**Review by carlesarnal — APPROVED** [review]
Minor tidy up comment. LGTM

File: distro/openshift-template/mt/tenant-manager-template.yaml (hunk 1 above)
Comment (andreaTP) [reply distro/openshift-template/mt/tenant-manager-template.yaml]: done 👍

**Comment by andreaTP** [comment]
Thanks @carlesarnal !

File: distro/openshift-template/mt/tenant-manager-template.yaml [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+- name: KEYCLOAK_REALM
+  description: Keycloak realm
+  value: "apicurio-registry"
+- name: KEYCLOAK_API_CLIENT_ID
+  description: client id for Apicurio Registry API
+  value: registry-api
+
+- name: TENANT_MANAGER_URL
+  value: http://tenant-manager:8585
+- name: TENANT_MANAGER_CA_BUNDLE
```
Comment (carlesarnal) [inline distro/openshift-template/mt/tenant-manager-template.yaml:351]: The other way around from the other PR. These env vars, related to the certificates and the auth client can be removed from here.

File: distro/openshift-template/mt/tenant-manager-template.yaml [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+  description: path to tenant-manager service CA bundle
+- name: TENANT_MANAGER_CLIENT_SECRET_NAME
+  description: k8s secret name with client-id and client-secret to access tenant-manager
+  required: true
+- name: TENANT_MANAGER_AUTH_TOKEN_EXP_REDUCTION_MS
```
Comment (carlesarnal) [inline distro/openshift-template/mt/tenant-manager-template.yaml:356]: Auth token exp can be removed as well.

File: distro/openshift-template/mt/tenant-manager-template.yaml [hunk 4]
```diff
@@ -0,0 +1,385 @@
...
+  description: Keycloak base url
+  value: "http://localhost:8090"
+- name: KEYCLOAK_REALM
+  description: Keycloak realm
+  value: "apicurio-registry"
+- name: KEYCLOAK_API_CLIENT_ID
+  description: client id for Apicurio Registry API
+  value: registry-api
+
+- name: TENANT_MANAGER_URL
```
Comment (carlesarnal) [inline distro/openshift-template/mt/tenant-manager-template.yaml:349]: This one can be removed as well.

**Review by carlesarnal — COMMENTED** [review]
More tidy-up comments. Sorry, I was too quick to approve.

File: distro/openshift-template/mt/tenant-manager-template.yaml (hunk 2 above)
Comment (andreaTP) [reply distro/openshift-template/mt/tenant-manager-template.yaml]: removed

File: distro/openshift-template/mt/tenant-manager-template.yaml (hunk 4 above)
Comment (andreaTP) [reply distro/openshift-template/mt/tenant-manager-template.yaml]: done

File: distro/openshift-template/mt/tenant-manager-template.yaml (hunk 3 above)
Comment (andreaTP) [reply distro/openshift-template/mt/tenant-manager-template.yaml]: done

**Comment by andreaTP** [comment]
@carlesarnal ready for next round 👍

**Review by carlesarnal — APPROVED** [review]

**Pull request merged**